README.md

```diff
@@ -75,7 +75,7 @@ to download only the polyfills needed by the current browser. Just add the
 following line to the start of your page:
```

Comment:
The description in this paragraph could be updated to be Cloudflare, https://cdnjs.cloudflare.com/polyfill